contracts/multisig-prover/src/encoding/evm.rs

```diff
@@ -55,6 +55,18 @@ impl TryFrom<Signer> for Operator {
     }
 }
 
+impl BatchID {
```

Comment:
why is this defined here, when the rest of BatchID is somewhere completely different